### ISSUE TITLE
receive sdo excute then response sdo

### ISSUE THREAD

**Issue by wangfandasha** [CANopenNode/CANopenNode#484]
I want to receive the sdo comand ，excute the command and then resonponse the relative command. what should i do if i want to use
 canopennode to realize this fuction?

**Comment by martinwag** [comment]
sdo command is handled internally by the stack. you can set a callback for when a sdo read/write is received, so that you can modify the data accordingly before it is applied / a response is sent.

**Comment by martinwag** [comment]
No. Either use a larger variable in your OD (e.g. octet string) or use Domain data type. Domain gives you a callback for start, data and end of transmission.
